cmds/modules/noded/twin.go

```diff
@@ -2,23 +2,33 @@ package noded
 
 import (
 	"context"
-
+	"crypto/ed25519"
+	"encoding/hex"
+	"fmt"
 	"github.com/pkg/errors"
-	"github.com/threefoldtech/go-rmb"
-	"github.com/threefoldtech/substrate-client"
+	"github.com/rs/zerolog/log"
+	"os"
+	"os/exec"
 )
 
-func runMsgBus(ctx context.Context, sub substrate.Manager, identity substrate.Identity) error {
-	// todo: make it argument or parse from broker
-	const redis = "/var/run/redis.sock"
-	app, err := rmb.NewServer(sub, redis, 100, identity)
-	if err != nil {
-		return err
-	}
+const (
+	redisAddr = "unix:///var/run/redis.sock"
```

Comment:
This is passed from the noded command line arguments, please don't use hard coded values